README.md

```diff
@@ -382,9 +382,25 @@ first_entry.fields('de-DE') # Returns German localizations
 
 - When an entry has related entries that are unpublished, they still end up in the resource as unresolved links. We consider this correct, because it is in line with the API responses and our other SDKs. However, you can use the workaround from [issue #60](/../../issues/60) if you happen to want this working differently.
 
-- While this library doesn't directly allow parsing Contentful webhook responses, you can check out [this code snippet](https://gist.github.com/neonichu/17a987aeeb256d4bf6f3) for a way to do it.
+## Migrating to 2.x
 
+If you're a `0.x` or a `1.x` user of this gem, and are planning to migrate to the current `2.x` branch.
+There are a few breaking changes you have to take into account:
+
+* `Contentful::Link#resolve` and `Contentful::Array#next_page` now require a `Contentful::Client` instance as a parameter.
+* `Contentful::CustomResource` does no longer exist, custom entry classes can now inherit from `Contentful::Entry` and have proper marshalling working.
+* `Contentful::Resource` does no longer exist, all resource classes now inherit from `Contentful::BaseResource`. `Contentful::Entry` and `Contentful::Asset` inherit from `Contentful::FieldsResource` which is a subclass of `Contentful::BaseResource`.
+* `Contentful::DynamicEntry` does no longer exist, if code checked against that base class, it should now check against `Contentful::Entry` instead.
+* `Contentful::Client#dynamic_entry_cache` _(private)_ has been extracted to it's own class, and can be now manually cleared by using `Contentful::ContentTypeCache::clear`.
+* `Contentful::BaseResource#sys` and `Contentful::FieldsResource#fields` internal representation for keys are now camel cased to match the instance accessors. E.g. `entry.fields[:myField]` previously had the accessor `entry.my_field`, now it is `entry.fields[:my_field]`. The value in both cases would correspond to the same field, only change is to unify the style. If code accessed the values through the `#sys` or `#fields` methods, keys now need to be camel cased.
```

Comment:
I'm not extra clear on this point, especially when you say both:
> keys now need to be camel cased

and 

>  E.g. `entry.fields[:myField]` previously had the accessor `entry.my_field`, now it is `entry.fields[:my_field]`

Does it mean that now everything needs to be camel cased, i.e. `entry.fields[:myField]` ? or That it needs NOT to be camel cased, i.e. `entry.fields[:my_field]`?